server/plugin_test.go

```diff
@@ -31,15 +32,20 @@ func TestPlugin(t *testing.T) {
 
 			str, _ := json.Marshal(user)
 
-			w.Write([]byte(str))
+			if _, err := w.Write(str); err != nil {
```

Comment:
nit: save the `if`, require will panic if err anyway

Reply:
How do you know this?